### ISSUE TITLE
Improvements to Features Page

### ISSUE THREAD

**Issue by coke11** [dragon-drop-scratch/html-app#1]
Here are some things that I would like added to the features page.

- [x] No longer have a big space in between the word features and the first thing - let's try to cut that in half

- [ ] Change wording on No Coding Knowledge Required (I will do that)

- [x] Center all of the text

- [ ] Change wording on support (I will do that)

- [x] Instead of you can always contact us why don't we make it look at what has been done with DragonDrop

- [x] Add an always free section

- [x] Add a no watermark section

- [x] Add a bring your own domain section

**Comment by csf30816** [comment]
Cool

**Comment by coke11** [comment]
You can do any of the task other than the ones that say I will do that

**Comment by csf30816** [comment]
ok (my favorite year is 1992)

**Comment by csf30816** [comment]
how can I view this site!!

**Comment by csf30816** [comment]
never mind

**Comment by csf30816** [comment]
:) they stuck with my original blue DD design

**Comment by csf30816** [comment]
Hows this? https://dragon-drop-scratch.github.io/html-app/features.html?1

**Comment by csf30816** [comment]
done with my stuff :)

**Comment by coke11** [comment]
Do you think we can put a line after features and before the first feature?

**Comment by coke11** [comment]
@csf30816 What do you use to design graphics?

**Comment by zspri** [comment]
We should make all of the graphics SVGs. (We don't have to, but it looks a lot better)

**Comment by coke11** [comment]
@Nanomotion I agree. Are all of yours svgs?

**Comment by csf30816** [comment]
Yeah it does! I actually converted my SVGs to PNGs for this project :P

**Comment by csf30816** [comment]
I use drawSVG

**Comment by coke11** [comment]
Do you think you could just upload the straight SVG's?

**Comment by zspri** [comment]
@coke11 Yes, I use Affinity Designer, which lets you use lots of different file formats

**Comment by coke11** [comment]
@Nanomotion Alright :D
